contracts/lacchain.system/ricardian/lacchain.system.contracts.md

```diff
@@ -0,0 +1,399 @@
+{{/if}}
+
+<h1 class="contract">setprods</h1>
+
+---
+spec_version: "0.2.0"
+title: Set Block Producers
+summary: 'Set block producer schedule'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+{{$action.account}} proposes a block producer schedule of:
+{{#each schedule}}
+  1. {{this.producer_name}}
+{{/each}}
+
+The block signing authorities of each of the producers in the above schedule are listed below:
+{{#each schedule}}
+### {{this.producer_name}}
+{{to_json this.authority}}
+{{/each}}
+
+<h1 class="contract">unlinkauth</h1>
+
+---
+spec_version: "0.2.0"
+title: Unlink Action from Permission
+summary: '{{nowrap account}} unsets the minimum required permission for the {{#if type}}{{nowrap type}} action of the{{/if}} {{nowrap code}} contract'
+icon: @ICON_BASE_URL@/@ACCOUNT_ICON_URI@
+---
+
+{{account}} removes the association between the {{#if type}}{{type}} action of the{{/if}} {{code}} contract and its minimum required permission.
+
+{{#if type}}{{else}}This will not remove any links explicitly associated to specific actions of {{code}}.{{/if}}
+
+<h1 class="contract">updateauth</h1>
+
+---
+spec_version: "0.2.0"
+title: Modify Account Permission
+summary: 'Add or update the {{nowrap permission}} permission of {{nowrap account}}'
+icon: @ICON_BASE_URL@/@ACCOUNT_ICON_URI@
+---
+
+Modify, and create if necessary, the {{permission}} permission of {{account}} to have a parent permission of {{parent}} and the following authority:
+{{to_json auth}}
+
+<h1 class="contract">onerror</h1>
+
+---
+spec_version: "0.2.0"
+title: On Error Action
+summary: 'This action is delivered to the sender of a deferred transaction'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+On error action, notification of this action is delivered to the sender of a deferred transaction
+when an objective error occurs while executing the deferred transaction.
+
+<h1 class="contract">addentity</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Entity
+summary: 'Add to the network a Partner and Non Partner entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Partner and Non Partner entity.
+
+<h1 class="contract">rmentity</h1>
+
+---
+spec_version: "0.2.0"
+title: Remove Entity
+summary: 'Remove from the network a Partner and Non Partner entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to remove from the network a Partner and Non Partner entity.
+
+<h1 class="contract">addvalidator</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Validator
+summary: 'Add to the network a Validator for the entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Validator for the entity.
+
+<h1 class="contract">addwriter</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Writer
+summary: 'Add to the network a Writer node for a Partner entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Writer node for a Partner entity.
```

Comment:
Allows a partner or non-partner entity to register a new writer node on the network.